### PR TITLE
Update yoga 3.0.4

### LINKED ISSUE #249: When will be support Yoga 3.0?

### PR DESCRIPTION
**Pull request by heoblitz**
## What is this PR for?

Update yoga to version 3.0.4

After this update, it will be possible to add support for position: static and align-content: space-evenly.
- https://www.yogalayout.dev/blog/announcing-yoga-3.0
- https://github.com/facebook/yoga/releases

## Tests
UnitTests ✅

Cocoapods, SPM Example projects build & run ✅

<br>

- close #249

### CODE REVIEW

File: FlexLayout.podspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |spec|
      '-fexceptions',
      '-Wall',
      '-Werror',
-     '-std=c++1y',
+     '-std=c++20',
```
Comment (heoblitz) [inline FlexLayout.podspec:22]: Update the compiler version to c++20 to support Yoga 3.

File: Sources/yoga/Algorithm_AbsoluteLayout.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include "Event_event.h"
```
Comment (heoblitz) [inline Sources/yoga/Algorithm_AbsoluteLayout.h:10]: Cocoapods does not support header/source files with same names and different paths, Using snake case to remove duplicate file names and flatten the folder structure.
- https://github.com/CocoaPods/CocoaPods/issues/8289

**Comment by heoblitz** [comment]
I will resolve the CI issue and add a commit.

File: Podfile.lock [hunk 3]
```diff
@@ -1,5 +1,5 @@
 PODS:
-  - FlexLayout (2.0.05)
+  - FlexLayout (2.0.07)
```
Comment (lucdion) [inline Podfile.lock:2]: 💅 
```suggestion
  - FlexLayout (2.0.7)
```

**Review by lucdion — APPROVED** [review]
Thanks for this important update

**Comment by lucdion** [comment]
The release 2.0.7 has been published, thanks for this change

**Pull request merged**